tools/astarte_export/lib/astarte/fetchdata/fetchdata.ex

```diff
@@ -285,9 +285,16 @@ defmodule Astarte.Export.FetchData do
     end
   end
 
-  defp from_native_type(value, :binaryblob) do
-    {:ok, binary_blob} = Base.encode64(value)
-    binary_blob
+  defp from_native_type(value, native_type) when is_list(value) do
+    case native_type do
+      :binaryblobarray -> Enum.map(value, &from_native_type(&1, :binaryblob))
+      :datetimearray -> Enum.map(value, &from_native_type(&1, :datetime))
+      _ -> Enum.map(value, &from_native_type(&1, native_type))
```

Comment:
If you call the `from_native_type/2` function with e.g. `from_native_type([1,2], :integerarray)`, wouldn't this line make the function call `from_native_type(1, :integerarray)`?
I would expect that the mapping between array and non-array type is preserved, as done for `binaryblobarray` and `datetimearray`